src/main.rkt

```diff
@@ -56,7 +56,7 @@
   (define demo-port 8000)
   (define demo-public #f)
 
-  (define threads #f)
+  (define threads (processor-count)) ; Or maybe 1 as the defualt?
```

Comment:
The default should definitely be 1 thread.

The old thread-pool supported a no-places fallback, where if you didn't ask for threads it simply wouldn't use threads. That was sometimes useful for debugging, because sometimes threads cause weird behavior with the FFI or other stuff. It would be nice to keep that feature.

I bet we could support no-threads mode by having `start-job` just save the command in a hash table and having `wait-for-job` run the job to completion. Perhaps let's implement that in this PR? Or what do you think?